### DISCUSSION TITLE
How to control where link inserts on child note when inserting two way link to main note

### DISCUSSION THREAD

**Discussion by Dcrizz** [windingwind/zotero-better-notes#884] [Q&A]
Hello

When inserting a two-way link to my main note, the link generated in the child always seems to be inserted at the bottom of the child note, not at the cursor position as it did in previous versions. Is there any way I can control this via the template?

**Comment by cloudwalke** [comment]
现在的双向链接必须要操作两次才能完成，要是能一次就插入一个双向链接就好了

**Comment by windingwind** [comment]
The link creator can locate the insert location at the begin/end of a section.
